### DISCUSSION TITLE
Is it ready "Adds support for Okta Verify TOTP MFA. [GH-10942] " ?

### DISCUSSION THREAD

**Discussion by pablo-gadi** [rajanadar/VaultSharp#271] [General]
Hello, 
          i would love to know when we could have "Adds support for Okta Verify TOTP MFA. [GH-10942]" ?

Thanks!

**Comment by rajanadar** [comment]
hi @pablo-gadi 

The support was added in Dec 2022.
https://github.com/rajanadar/VaultSharp/blob/master/CHANGELOG.md#1812-december-09-2022
